extensions/pom.xml

```diff
@@ -200,8 +200,8 @@
                     <artifactIdPrefix>quarkus-</artifactIdPrefix>
                     <namePrefix xml:space="preserve">Quarkus - </namePrefix>
                     <quarkusVersion>@{project.version}</quarkusVersion>
-                    <runtimeBomPath>../bom/runtime/pom.xml</runtimeBomPath>
-                    <deploymentBomPath>../bom/deployment/pom.xml</deploymentBomPath>
+                    <runtimeBomPath>../bom/1pp/pom.xml</runtimeBomPath>
```

Comment:
Is this path correct? `1pp`?

Reply:
Also shouldn't it be `../bom/application/pom.xml` for both?

Reply:
omg, thanks

Reply:
how did it not fail?.. Maven is weird.

Reply:
This part seems to be commented out, maybe that's why?

Reply:
Ah, right, it's commented out. Yes, it should be the same path. I'll just update the plugin to remove the deploymentBomPath.